tests/test_cli_server_client_gcp.py

```diff
@@ -0,0 +1,128 @@
+"""This does not cover all CLI parameters defined in caper/caper_args.py.
+gcp (Google Cloud Platform) backend is tested here with server/client functions.
+"""
+import os
+import time
+
+from caper.cli import main as cli_main
+from caper.cromwell_rest_api import CromwellRestAPI
+from caper.wdl_parser import WDLParser
+
+from .example_wdl import make_directory_with_wdls
+
+TIMEOUT_SERVER_SPIN_UP = 500
+TIMEOUT_SERVER_RUN_WORKFLOW = 960
+
+
+def test_server_client(
```

Comment:
I'd like to see a similar mark to indicate test use of Google Cloud like I mentioned in `test_cli_run.py`:
```suggestion
@pytest.mark.google_cloud
@pytest.mark.integration
def test_server_client(
```

Reply:
Added it as you suggested.